src/Catalog/NuGet.Services.Metadata.Catalog.csproj

```diff
@@ -320,13 +317,13 @@
       <Version>5.0.0-preview1.5665</Version>
     </PackageReference>
     <PackageReference Include="NuGet.Services.Logging">
-      <Version>2.25.0-master-30088</Version>
+      <Version>2.57.0</Version>
     </PackageReference>
     <PackageReference Include="System.Net.Http">
       <Version>4.3.3</Version>
     </PackageReference>
     <PackageReference Include="WindowsAzure.Storage">
-      <Version>8.2.1</Version>
+      <Version>9.3.3</Version>
```

Comment:
NuGet.Services.Storage depends on WindowsAzure.Storage >= 9.3.3. This aligns with that version constraint.